src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php

```diff
@@ -145,4 +148,26 @@ protected function templateExists($template)
 
         return false;
     }
+
+    private function processLogs($logger)
+    {
+        $logs = array();
+        foreach ($logger->getLogs() as $log) {
+            if (class_exists('Symfony\Component\VarDumper\Dumper\HtmlDumper')) {
+                $cloner = new VarCloner();
+                $dumper = new HtmlDumper();
+                $dumper->dump($cloner->cloneVar($log['context']), $output = fopen('php://memory', 'r+b'));
```

Comment:
You can just pass `true` as the second argument since 3.2 (https://github.com/symfony/symfony/commit/4be977627edce4e894f496921280f1092241025d) to get the string as returned value.